### DISCUSSION TITLE
Grid view option for Articles

### DISCUSSION THREAD

**Discussion by goliv04053** [jocmp/capyreader#304] [Feature Requests]
I have two devices. A smartphone and a tablet. I use Capy on both. I also was testing a feed reader named Feeder on my tablet and I seen this function there. I have various feeds so I see them almost all at once and have a larger view to choose is nice.

I think that it would be interesting to see it also implemented also on Capy Reader.

**Comment by jocmp** [comment]
I tried out Feeder's grid view on my tablet. The major difference between Feeder and Capy on tablets is that Capy uses a list-detail view while Feeder presents the list and article on separate pages.

Is your idea that the list would stay the same size but with a grid option, or would it be fullscreen like Feeder? It's relatively easy to add a grid option to the list-detail view in Capy but would require a lot of to make it fullscreen.

![article_light](https://github.com/user-attachments/assets/6e492b69-5ba6-4b4f-99d8-80e3380c992c)

**Reply by goliv04053** [reply]
In the long run, i think it would be interesting as the way applied on Feeder. In the short term, i think it would be fine to have an option to add grid view on list-detail. I just got curious about how it would look in this second option. 

Not related, but do you have plans to have a GitHub Sponsor/ko-fi/Patreon enabled to the project? I mean, to help fund development to Capy Reader.

**Reply by jocmp** [reply]
It might look tight in the current list-detail view but I think I'll explore more layout options at some point in the future along with some other planned customization updates.

I have both a GitHub sponsor setup (https://github.com/sponsors/jocmp/) and Ko-fi (https://ko-fi.com/capyreader).
